README.md

```diff
@@ -20,6 +20,7 @@ packages
 ├── ui5-middleware-livetranspile    // middleware extension: on-demand es6 to es5 transpile when requesting js
 ├── ui5-middleware-simpleproxy      // middleware extension: simple express proxy
 ├── ui5-middleware-servestatic      // middleware extension: serve static resources
+├── ui5-task-flatten-library        // task extension: Prepares build result for deployment to SAP NetWeaver
```

Comment:
please move 1 line down (alphabetical sorting of tasks) and lower-case "p" in "Prepares"

Reply:
I'd say `f` comes before `i`, right?

Reply:
😪 that's right...